compute/model.go

```diff
@@ -520,7 +520,7 @@ type JobEmailNotifications struct {
 type CronSchedule struct {
 	QuartzCronExpression string `json:"quartz_cron_expression"`
 	TimezoneID           string `json:"timezone_id"`
-	PauseStatus          string `json:"pause_status,omitempty" tf:"computed"`
```

Comment:
What happens if you keep tf computed flag? I'd rather keep the schema compatible

Reply:
removed